awx/locale/django.pot

```diff
@@ -791,7 +791,7 @@ msgid "Request to {} timed out."
 msgstr ""
 
 #: awx/api/views.py:2129
```

Comment:
@shanemcd correct me if I'm wrong - changes to this `.pot` file should have corresponding changes in the `.py` source code files they reference.

Reply:
sorry, let me add one more fix if that is OK ?

Reply:
This file should not be updated manually. It is generated from the source code. There are arguments for this file not being in source control at all, but that is a separate discussion.

Reply:
my mistake.  Would this string change make sense then ?
```
diff --git a/awx/ui/conf.py b/awx/ui/conf.py
index 0d626c28f..577b9898f 100644
--- a/awx/ui/conf.py
+++ b/awx/ui/conf.py
@@ -58,7 +58,7 @@ register(
     field_class=fields.IntegerField,
     min_value=100,
     label=_('Max Job Events Retreived by UI'),
-    help_text=_('Maximum number of job events for the UI to retreive within a '+    help_text=_('Maximum number of job events for the UI to retrieve within a '                 'single request.'),
     category=_('UI'),
```

Reply:
@jeis2497052 yep, if you don't mind, feel free to update the related `.py` files like in your example here.  But let's leave the `.pot` file unchanged - it's modified via external tooling.